reportService/reportsServer.js

```diff
@@ -66,7 +66,8 @@ const PAGE_MARGIN = 60;
   const headerRightImage = process.argv[9] || '';
   const pageSize = process.argv[11] || PAGE_SIZES.Letter;
   const disableHeaders = process.argv[12] === true || process.argv[12] === "true";
-  const chromeExecution = process.argv[13] || paths['chromium'] || paths['google-chrome-stable'] || paths['google-chrome'] || '/usr/bin/chromium-browser';
+  const forceAutoHeightLayout = process.argv[13] === true || process.argv[13] === "true";
```

Comment:
Why did you change the order? Why not adding the new one as 14? This will break compatibility.

Reply:
i think we can assume `chromeExecution` is not used yet (only by the server) and the `forceAutoHeightLayout` should be before. i can switch it if we fear bc

Reply:
Let's switch.